x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/activity_monitor/index.tsx

```diff
@@ -0,0 +1,337 @@
+    {
+      id: 13,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 14,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 15,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 16,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 17,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 18,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 19,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 20,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 21,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+  ];
+
+  const [itemsTotalState] = useState<number>(sampleTableData.length);
+  const [pageState, setPageState] = useState<PageTypes>({ index: 0, size: 20 });
+  // const [selectedState, setSelectedState] = useState<ColumnTypes[]>([]);
```

Comment:
You want this commented out code to live?

Reply:
As above, everything under `activity_monitor/` is all the initial placeholder content added as part of https://github.com/elastic/kibana/pull/50220, so it'll be going away when it's time to implement that component.